src/test/java/nextstep/StringTest.java

```diff
@@ -0,0 +1,90 @@
+package nextstep;
+
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import java.util.stream.Stream;
+
+public class StringTest {
+
+
+    @ParameterizedTest
+    @MethodSource("provideStringForSplitTest")
+    public void splitTest(String parameter, String[] expected) {
+
+        // when
+        String[] splitted = parameter.split(",");
+
+        // then
+        Assertions.assertThat(splitted).contains(expected);
+        Assertions.assertThat(splitted).containsExactly(expected);
```

Comment:
```suggestion
        Assertions.assertThat(splitted).containsExactly(expected);
```

하나의 assert로 두 가지 개념을 같이 검증할 수 있을 것 같아요 :)